entity-service-impl/src/main/java/org/hypertrace/entity/service/util/DocStoreConverter.java

```diff
@@ -183,6 +185,23 @@ private static Filter transformToOrFilterChainForStrArray(AttributeFilter attrib
     return f;
   }
 
+  private static Filter transformToAndFilterChainForStrArray(AttributeFilter attributeFilter) {
```

Comment:
What about the fact the attribute itself might not exist? This would only work for the case, where the attribute exists

I think we would also want a NOT_EXISTS filter OR `transformToAndFilterChainForStrArray`

Reply:
Yes, based on `NEQ` impl here, we need `exits` condition in consumers. Added.